turbinia/workers/__init__.py

```diff
@@ -18,16 +18,21 @@
 import json
 import logging
 import os
+import platform
 import time
 import uuid
 
```

Comment:
Yeah, probably a good idea.  I added it here and in output_writers.py.  I'll go through with a second PR to add it elsewhere.